src/main/java/seedu/lovebook/logic/commands/BestMatchCommand.java

```diff
@@ -0,0 +1,21 @@
+package seedu.lovebook.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.lovebook.model.Model;
+
+/**
+ * Finds the best match based on date preferences of the user.
+ */
+public class BestMatchCommand extends Command {
+    public static final String COMMAND_WORD = "bestMatch";
```

Comment:
would be better to just use `match` instead of `bestMatch` imo. [Just a suggestion]